core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.controller.ts

```diff
@@ -111,17 +114,12 @@ angular.module('oppia').controller('QuestionSuggestionEditorModalController', [
     };
     $scope.skillId = $scope.skill.getId();
     $scope.onClickChangeDifficulty = function() {
-      $uibModal.open({
-        templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
-          '/pages/topic-editor-page/modal-templates/' +
-          'select-skill-and-difficulty-modal.template.html'),
-        backdrop: true,
-        resolve: {
-          skillId: () => $scope.skillId
-        },
-        controller: (
-          'QuestionsOpportunitiesSelectSkillAndDifficultyModalController')
-      }).result.then(function(result) {
+      let modalRef: NgbModalRef = NgbModal.open(
```

Comment:
Nit: If the variable is not being reassigned, can declare as `const`. Here and in core/templates/pages/contributor-dashboard-page/question-opportunities/question-opportunities.component.ts.

Reply:
Done.